<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/ico/icon</normaloff>:/ico/icon</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableView" name="tableView"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>769</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="new1"/>
    <addaction name="open"/>
    <addaction name="separator"/>
    <addaction name="save"/>
    <addaction name="saveAs"/>
    <addaction name="separator"/>
    <addaction name="action_4"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Правка</string>
    </property>
    <addaction name="add"/>
    <addaction name="separator"/>
    <addaction name="cut"/>
    <addaction name="copy"/>
    <addaction name="paste"/>
    <addaction name="separator"/>
    <addaction name="append"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Поиск</string>
    </property>
    <addaction name="find"/>
    <addaction name="findNext"/>
    <addaction name="findPrev"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="FAQ"/>
    <addaction name="separator"/>
    <addaction name="about"/>
    <addaction name="aboutQt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>24</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="new1"/>
   <addaction name="open"/>
   <addaction name="save"/>
   <addaction name="separator"/>
   <addaction name="add"/>
   <addaction name="copy"/>
   <addaction name="paste"/>
   <addaction name="cut"/>
   <addaction name="separator"/>
   <addaction name="find"/>
  </widget>
  <action name="open">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
   </property>
   <property name="text">
    <string>Открыть</string>
   </property>
   <property name="toolTip">
    <string>Открытие файла</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="save">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
   <property name="toolTip">
    <string>Сохранение файла</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="add">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/add.png</normaloff>:/images/add.png</iconset>
   </property>
   <property name="text">
    <string>Добавить данные</string>
   </property>
   <property name="toolTip">
    <string>Добавить данные о машине в базу данных</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="copy">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/copy2.png</normaloff>:/images/copy2.png</iconset>
   </property>
   <property name="text">
    <string>Копировать</string>
   </property>
   <property name="toolTip">
    <string>Копировать данные</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="paste">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/paste.png</normaloff>:/images/paste.png</iconset>
   </property>
   <property name="text">
    <string>Вставить</string>
   </property>
   <property name="toolTip">
    <string>Вставить данные</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="cut">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/cut.png</normaloff>:/images/cut.png</iconset>
   </property>
   <property name="text">
    <string>Вырезать</string>
   </property>
   <property name="toolTip">
    <string>Вырезать данные</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="find">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/find.png</normaloff>:/images/find.png</iconset>
   </property>
   <property name="text">
    <string>Поиск</string>
   </property>
   <property name="toolTip">
    <string>Поиск данных по заданным критериям</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="new1">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/New.png</normaloff>:/images/New.png</iconset>
   </property>
   <property name="text">
    <string>Новый файл</string>
   </property>
   <property name="toolTip">
    <string>Создать новый файл</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="saveAs">
   <property name="text">
    <string>Сохранить как...</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="findNext">
   <property name="text">
    <string>Найти следующее</string>
   </property>
  </action>
  <action name="findPrev">
   <property name="text">
    <string>Найти предыдущее</string>
   </property>
  </action>
  <action name="FAQ">
   <property name="text">
    <string>Справка</string>
   </property>
  </action>
  <action name="about">
   <property name="text">
    <string>О программе..</string>
   </property>
  </action>
  <action name="aboutQt">
   <property name="text">
    <string>О Qt</string>
   </property>
  </action>
  <action name="append">
   <property name="text">
    <string>Объединить таблицы</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>tableView</tabstop>
 </tabstops>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>tableView</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>MainWindow</receiver>
   <slot>pushSelect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>343</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>383</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>pushSelect()</slot>
  <slot>Excel()</slot>
 </slots>
</ui>
